### ISSUE TITLE
clang-format: Failed to download resource clang-format_bottle_manifest

### ISSUE THREAD

**Issue by peaceiris** [Homebrew/linuxbrew-core#23906]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
$ brew config
HOMEBREW_VERSION: 3.2.3
ORIGIN: https://github.com/Homebrew/brew
HEAD: 16b01d6e019e93b66e41d0bb8da66fec3fcc5cca
Last commit: 3 days ago
Core tap ORIGIN: https://github.com/Homebrew/linuxbrew-core
Core tap HEAD: 218a4081cb07ca96d4fd4fa466cae4fbbc42f987
Core tap last commit: 89 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: localhost:10.0
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.3 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
CPU: quad-core 64-bit skylake
Clang: N/A
Git: 2.32.0 => /home/linuxbrew/.linuxbrew/bin/git
Curl: 7.68.0 => /usr/bin/curl
Kernel: Linux 5.4.0-77-generic x86_64 GNU/Linux
OS: Ubuntu 20.04.2 LTS (focal)
Host glibc: 2.31
/usr/bin/gcc: 9.3.0
/usr/bin/ruby: N/A
glibc: N/A
gcc@5: N/A
xorg: N/A
$ brew doctor
Your system is ready to brew.
```

### 

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

`brew install clang-format`

### What happened (include all command output)?

```console
$ brew install clang-format
==> Downloading https://ghcr.io/v2/linuxbrew/core/clang-format/manifests/12.0.1
curl: (22) The requested URL returned error: 404                              

Error: Failed to download resource "clang-format_bottle_manifest"
Download failed: https://ghcr.io/v2/linuxbrew/core/clang-format/manifests/12.0.1
```

### What did you expect to happen?

Installing clang-format finishes successfully.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
$ brew update && brew outdated && brew upgrade
Already up-to-date.
$ brew install clang-format
==> Downloading https://ghcr.io/v2/linuxbrew/core/clang-format/manifests/12.0.1
curl: (22) The requested URL returned error: 404                              

Error: Failed to download resource "clang-format_bottle_manifest"
Download failed: https://ghcr.io/v2/linuxbrew/core/clang-format/manifests/12.0.1
```

**Comment by joelberkeley** [comment]
i'm also seeing the same for `brew install idris2`

**Comment by joelberkeley** [comment]
also similar to https://github.com/Homebrew/linuxbrew-core/issues/23713 do we know if these are the same problem?

**Comment by brettcannon** [comment]
I'm seeing the same issue for `python-launcher`.

**Comment by joelberkeley** [comment]
does anyone have any idea how to fix this kind of issue? I'd be happy to try but I've no idea where to start

**Comment by brettcannon** [comment]
@joelberkeley My guess is some GitHub Action workflow that generates that manifest file failed (either due to a bug in the workflow or a hiccup on GitHub's side, e.g. GHR having problems at that exact moment). Probably the best option is to try and find the GitHub Action which creates the manifest files and then try to find the workflow run that was _supposed_ to create the manifest file and failed (or find out in fact that workflow run never existed to begin with).

I found https://github.com/Homebrew/homebrew-core/pull/80982 as the PR on homebrew-core which I assume is what Linuxbrew picked up.

**Comment by branchvincent** [comment]
i'm not sure the expected behavior here, but it's trying to pull the bottle from [`https://ghcr.io/v2/linuxbrew`](https://github.com/orgs/Linuxbrew/packages) whereas it only exists right now at [`https://ghcr.io/v2/homebrew`](https://github.com/Homebrew/homebrew-core/pkgs/container/core%2Fpython-launcher).

as `homebrew-core` is periodically merged into `linuxbrew-core`, it [looks like](https://github.com/Homebrew/linuxbrew-core/commit/b9fe8fbf246db49d3e2a4251428d23c1bc537f64#diff-d8001ee58d93642086f52df35a31be6f227434e7fb46cffdc53a1919578e723f) the homebrew-built bottle is meant to be manually removed so it can be [`rebuilt`](https://github.com/Homebrew/linuxbrew-core/blob/master/.github/workflows/request-bottle-after-merge.yml) here. doesn't look like that happened with these formulae ([here](https://github.com/Homebrew/linuxbrew-core/runs/3037480293) is the failure for `clang-format`)

**Comment by joelberkeley** [comment]
i'll be honest I've just found a workaround and after a week or so of installation problems I want a break from debugging. I may come back to this, at least in part because it's faster than my current approach, but I don't know when

**Comment by iMichka** [comment]
Fixed by 3726a13f0a1763df81b5f00d53ee88686f45edca
